openfisca_aotearoa/tests/income_tax/family_scheme/best_start.yaml

```diff
@@ -141,12 +162,19 @@
       2018-07: [0, 0, 0, 260 ]
     best_start__entitlement:
       2018-07: [260, 0, 0, 0]
+
 - name: Tests paid parental leave entitlement value with baby due date after 1st July 2018
   period: 2018
   absolute_error_margin: 0
   persons:
     - id: "Papatūānuku"
       date_of_birth: 1979-02-01
+      social_security__is_ordinarily_resident_in_new_zealand:
```

Comment:
I can see this added to the tests, but I can't fun where you've added this to the PPL code.

How about a test for the scenrior where `social_security__is_ordinarily_resident_in_new_zealand` is false.

Also a reference. Where in the PPL act does it require the social security act definition of being resident in NZ.

I'm reviewing on a small screen of my phone so sorry if I've missed it.

Reply:
It appears the tests were named incorrectly, they are nothing to paid parental leave - but rather best start entitlement (the file is named correctly).
I have now also added a test for false as suggested.

Reply:
So the Income Tax Act is using the Social Security Act definition of ordinarily resident in NZ?
Is there a reference URL? I thought Income Tax act has it's own residence definition.

Reply:
I've got it wrong :-(. It was instructed that we use this social security definition, but after a re-investigation I'll need to revert these particular changes and instead advise the smartstart team to utilise the `income_tax__residence` variable.